### PR TITLE
18668 - External influence section on add SI component.

### PR DESCRIPTION
**Pull request by hfekete**
*Issue:18668 *https://github.com/bcgov/entity/issues/18668 

*Description of changes:*
- Added section for "external influence" on add individual page
- added info on summary table 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-110-7rvwfzek.web.app

File: btr-web/btr-main-app/components/individual-person/SummaryTable.vue [hunk 1]
```diff
@@ -105,6 +111,21 @@
...
         </template>
       </tr>
+      <!-- standard class or css style without !important was not working -->
+      <tr
+        v-if="displayAdditional(item) && editingIndex != index"
+        style="border-top-width: 0!important"
+        data-cy="summary-table-external-influence"
+      >
+        <td colspan="6">
+          <p v-if="item.externalInfluence === ExternalInfluenceE.CAN_BE_INFLUENCED">
```
Comment (kialj876) [inline btr-web/btr-main-app/components/individual-person/SummaryTable.vue:121]: The text you're showing here doesn't match the design ticket fyi

**Review by kialj876 — COMMENTED** [review]
Hey hrvoje, I have a few comments / questions. Also, I don't see any code to add this to the submission when it gets converted to bods and back? I think this data will just be lost on submission currently

File: btr-web/btr-main-app/components/individual-person/AddNew.vue [hunk 2]
```diff
@@ -206,6 +206,47 @@
...
+          <label for="radio-can-influence" class="ml-5">
+            <BcrosI18HelperBold translation-path="labels.externalInfluence.canInfluence" />
+          </label>
+        </div>
+        <div class="flex items-center mb-2 py-1">
+          <URadio
+            id="radio-can-be-influenced"
+            v-model="significantIndividual.externalInfluence"
+            :value="ExternalInfluenceE.CAN_BE_INFLUENCED"
+            data-cy="external-influence-radio-can-be-influenced"
```
[the remark this answers is not in the document]
Comment (hfekete) [reply btr-web/btr-main-app/components/individual-person/AddNew.vue]: Re-ordered them

File: btr-web/btr-main-app/components/individual-person/SummaryTable.vue [hunk 3]
```diff
@@ -6,7 +6,10 @@
     :empty-state="$t('texts.tables.emptyTexts.individualsSummaryTable')"
   >
     <template #table-row="{ item, index }">
-      <tr v-if="item.action != FilingActionE.REMOVE && editingIndex != index">
+      <tr
+        v-if="item.action != FilingActionE.REMOVE && editingIndex != index"
+        class="border-0"
```
[the remark this answers is not in the document]
Comment (hfekete) [reply btr-web/btr-main-app/components/individual-person/SummaryTable.vue]: yeah, you are right, this one is leftover

File: btr-web/btr-main-app/components/individual-person/SummaryTable.vue (hunk 1 above)
Comment (hfekete) [reply btr-web/btr-main-app/components/individual-person/SummaryTable.vue:118]: This was previously  also a text of radio buttons, the thing is the text changed, but I think summary table one is not updated.
Will need to check with Scott on that one, so lets UX doublecheck that one.

Comment (hfekete) [reply btr-web/btr-main-app/components/individual-person/SummaryTable.vue:118]: Yeah, Scott confirmed.

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-110-7rvwfzek.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-110-7rvwfzek.web.app

**Review by kialj876 — APPROVED** [review]

**Pull request merged**